### PR TITLE
Stop using deprecated (and removed) Chrome DOM API

### PR DESCRIPTION
**Pull request by braxnu**
This fixes errors caused by changes to Chrome DOM API related to this change https://code.google.com/p/chromium/issues/detail?id=504574

### CODE REVIEW

**Comment by braxnu** [comment]
@Mendeley/developers

File: src/Schema.js [hunk 1]
```diff
@@ -337,7 +337,7 @@ define(['src/options', 'src/storage', 'src/debug'], function (CSLEDIT_options, C
 
 			// add child results to the result list
 			$.each(node.childNodes, function (i, childNode) {
-				if (childNode.localName !== null) {
+				if (childNode.nodeName !== '#text') {
```
Comment (paulwib) [inline src/Schema.js:340]: Is this testing if `childNode` is a text node or not? Why not test `nodeType` like below?

Comment (braxnu) [inline src/Schema.js:340]: I've got biased by surrounding lines. This is actually a test for non text node (we don't expect other types here). Could equally be `nodeType === 1`

Comment (paulwib) [inline src/Schema.js:340]: :+1:

**Pull request merged**